website/pages/docs/connect/wan_federation_via_mesh_gateways.mdx

```diff
@@ -0,0 +1,217 @@
+---
```

Comment:
I don't think this file should be underscore separated, I think it should be dash separated  to match the rest  of the docs